stdlib/http/src/test/java/org/ballerinalang/stdlib/services/nativeimpl/request/RequestNativeFunctionSuccessTest.java

```diff
@@ -748,4 +748,16 @@ public void testGetCookies() {
                            "No cookie objects in the Return Values");
         Assert.assertTrue(returnVals.length == 1);
     }
+
+    @Test
+    public void testGetCookiesWithEmptyValue() {
+        ObjectValue inRequest = createRequestObject();
+        ObjectValue entity = createEntityObject();
+        inRequest.set(REQUEST_ENTITY_FIELD, entity);
+        BValue[] returnVals = BRunUtil.invoke(compileResult, "testGetCookiesWithEmptyValue",
+                new Object[]{inRequest});
+        Assert.assertFalse(returnVals.length == 0 || returnVals[0] == null,
+                "No cookie objects in the Return Values");
+        Assert.assertEquals(returnVals.length, 1);
```

Comment:
Isn't this assertion already implied by the previous assertion ?

Reply:
No, here, we are checking the actual length of the output but the previous assertion is checking that the output and output's first value are not empty.